src/SerilogWeb.Classic/Classic/WebApiExceptionLogger.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web;
+using System.Web.Http;
+using System.Web.Http.ExceptionHandling;
+
+namespace SerilogWeb.Classic
+{
+    /// <summary>
+    /// Add the following line in Global.asax.cs
+    /// to log WebApi faults
+    /// <code>
+    /// protected void Application_Start()
+    ///   {
+    ///       GlobalConfiguration.Configure(SerilogWeb.Classic.WebApiExceptionLogger.Register);
+    ///   }
+    /// </code>
+    /// </summary>
+    public class WebApiExceptionLogger : IExceptionLogger
+    {
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public static Action<HttpConfiguration> Register
+            => c => c.Services.Add(typeof (IExceptionLogger),
+                new WebApiExceptionLogger());
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="context"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public Task LogAsync(ExceptionLoggerContext context, CancellationToken cancellationToken)
```

Comment:
There's a non-async method available to override too - Log().  Since no async work is done would that be better?